src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -0,0 +1,385 @@
+			const w = el.offsetWidth
+			if (hide) {
+				el.classList.add('hidden')
+			}
+			return w
+		},
+		cancelEvent(e) {
+			e.stopPropagation()
+			e.preventDefault()
+			return false
+		},
+		dragstart(e) {
+			return this.cancelEvent(e)
+		},
+		dropped(path, e) {
+			this.cancelEvent(e)
+			this.$emit('dropped', path)
+			this.$refs.actions.opened = false
+			const crumbs = document.querySelectorAll('.crumb')
+			crumbs.forEach((f) => { f.classList.remove('over') })
+			return false
+		},
+		dragOver(e) {
+			if (e.preventDefault) {
+				e.preventDefault()
+			}
+			return false
+		},
+		dragEnter(e) {
+			// Get the correct element, in case we hover a child.
+			if (e.target.closest) {
+				const target = e.target.closest('.crumb')
+				if (target.classList && target.classList.contains('crumb')) {
+					const crumbs = document.querySelectorAll('.crumb')
+					crumbs.forEach((f) => { f.classList.remove('over') })
+					target.classList.add('over')
+				}
+			}
+		},
+		dragLeave(e) {
+			// Don't do anything if we leave towards a child element.
+			if (e.target.contains(e.relatedTarget)) {
+				return
+			}
+			// Get the correct element, in case we leave directly from a child.
+			if (e.target.closest) {
+				const target = e.target.closest('.crumb')
+				if (target.contains(e.relatedTarget)) {
+					return
+				}
+				if (target.classList && target.classList.contains('crumb')) {
+					target.classList.remove('over')
+				}
+			}
+		},
+		openActions() {
+			this.$refs.actions.opened = true
+		}
+	},
+	render: function(createElement) {
+		// Get the breadcrumbs
+		const breadcrumbs = (this.$slots.default || []).filter(node =>
+			!!node
+			&& !!node.componentOptions
+			&& node.componentOptions.tag === 'Breadcrumb'
+		)
+
+		// Check that we have at least one breadcrumb
+		if (breadcrumbs.length === 0) {
+			return
+		}
+
+		// Add the root icon to the first breadcrumb
+		Vue.set(breadcrumbs[0].componentOptions.propsData, 'icon', this.rootIcon)
+
+		const crumbs = []
+		// The first half of the breadcrumbs
+		const crumbs1 = this.hiddenIndices.length
+			? breadcrumbs.slice(0, Math.round(breadcrumbs.length / 2))
+			: breadcrumbs
+		// Todo remove duplicate code
+		crumbs1.forEach((crumb, i) => {
+			if (crumb.elm && crumb.elm.classList) {
+				if (this.isHidden(i)) {
+					crumb.elm.classList.add('hidden')
+				} else {
+					crumb.elm.classList.remove('hidden')
+				}
+			}
+			crumbs.push(crumb)
+		})
+
+		// The Actions menu
+		if (this.hiddenCrumbs.length) {
+			crumbs.push(createElement('div', { class: 'crumb' },
+				[createElement(Actions, {
+					class: 'dropdown',
+					props: {
+						forceMenu: true
+					},
+					ref: 'actions',
+					nativeOn: {
+						dragstart: this.dragstart,
+						dragenter: this.openActions,
+						dragleave: this.closeActions
+					}
+				}, this.hiddenCrumbs.map(crumb => {
+					const path = crumb.componentOptions.propsData.path
+					return createElement(ActionRouter, {
```

Comment:
I can also be `ActionLink` if there is no `:to`, right? :thinking:

Reply:
Shouldn't there always be a `:to`?

Reply:
If it's a simple link, no :)
Not every apps uses vue-router ;)

Reply:
Hm, how should we decide if it is an ActionLink or ActionRouter then? They both need a path/href.

Reply:
Maybe have to props: `href` and `to`. `to` makes it a ActionRouter, `href` will lead to an `ActionLink`. This should fix the netlifiy issue not rendering `router-link` as well.

Reply:
I added two props now, `to` and `href`. When `to` is set, it uses `router-link` and `ActionRouter`, if not, it uses `a` and `ActionLink`.